x-pack/agent/_meta/agent.yml

```diff
@@ -8,12 +8,21 @@ outputs:
     username: elastic
     password: changeme
 
-streams:
-  - type: event/file
-    enabled: true
-    paths:
-      - /var/log/hello1.log
-      - /var/log/hello2.log
+datasources:
+  - namespace: sample
+    use_output: default
+    inputs:
+      - type: logs
+        processors:
+        streams:
+          - dataset: sample.acccess
```

Comment:
I suggest if we don't have any good default logs here, we just skip the log parts / leave it commented out. I would suggest the system logs here (check logs system module)

Reply:
system module seems purely linux oriented so i left it out so our future windows test wont fail on something like this

Reply:
That is where in the future our constraints come into play. I think part of it work also on OS X

The samples you put in will also not work. Suggestion: Lets skip the logs part for now but have a logs sample based on the Linux logs inside but commented out. So users can easily edit it.